lmdeploy/model.py

```diff
@@ -18,6 +18,15 @@ def random_uuid() -> str:
     return str(uuid.uuid4().hex)
 
 
+def get_text(content: Union[str, List[dict]]):
+    """openai format support Union[str,
+    List[ChatCompletionContentPartTextParam]] input."""
```

Comment:
may get updated

Reply:
@irexyc May update the docstring as follows:
"""Within the OpenAI API, the content field may be specified as either a string or a list of ChatCompletionContentPartTextParam (defined in openai). When a list is provided, lmdeploy selects the first element to incorporate into the chat template, as the manner in which OpenAI processes lists is not explicitly defined.
"""